js/templates/components/pageControlsTextStyle.html

```diff
@@ -0,0 +1,35 @@
+<%
+let countsAvailable;
+
+if (typeof ob.start === 'number' &&
```

Comment:
Do we really need `start`, `end`, `total` and `currentPage` page?

Wouldn't `currentPage` and a `morePages` bool be enough?